### PR TITLE
Update API, improve web ui, add chart and improvements in Android app.

### PR DESCRIPTION
**Pull request by 4o4fra**
This pull request includes several changes to the Android app and the API, focusing on notification enhancements, settings updates, database improvements, and code refactoring. The most important changes are summarized below:

### Android App Enhancements:

* [`PlutaReminderReceiver.kt`](diffhunk://#diff-a92267d3fa0a027f665eebfdbeec0e0977a10f7780cf5c7c46ca4c5a52caa08eR3-R29): Added functionality to launch `MainActivity` when a notification is clicked, using `PendingIntent` to handle the intent.
* [`SettingsActivity.kt`](diffhunk://#diff-6540ce81d69010a5bfb8cfbb491a1e7f9e57ba1e93bb768b2808d296afe21db7R3-R9): Introduced new settings options, including changing the nickname and an Easter egg feature that appears after tapping a button multiple times. [[1]](diffhunk://#diff-6540ce81d69010a5bfb8cfbb491a1e7f9e57ba1e93bb768b2808d296afe21db7R3-R9) [[2]](diffhunk://#diff-6540ce81d69010a5bfb8cfbb491a1e7f9e57ba1e93bb768b2808d296afe21db7R18-L25) [[3]](diffhunk://#diff-6540ce81d69010a5bfb8cfbb491a1e7f9e57ba1e93bb768b2808d296afe21db7L37-R43) [[4]](diffhunk://#diff-6540ce81d69010a5bfb8cfbb491a1e7f9e57ba1e93bb768b2808d296afe21db7R80-R115)

### API Database Improvements:

* [`db.ts`](diffhunk://#diff-abffd3c724890d47dcd4b487b3997ea8d9f2ff9b861888f6555bf287a050c460L5-R5): Updated the database path to ensure it resolves correctly.
* [`initDb.ts`](diffhunk://#diff-cd9019c85bee945fc42cb195eff62614ef09293e847f71f35d71073789de06b6R31-R36): Added a new table `pluta_log` to store Pluta values with timestamps.
* [`plutaLogDb.ts`](diffhunk://#diff-2041ed29f3e3d0bac450bb9d675e6ac685cbdb4e09cfcaf412fb0a33c366ab48R1-R24): Created functions to save Pluta values to the database and retrieve logs, enhancing data persistence and retrieval.

### API Refactoring:

* Renamed and refactored several utility functions and modules to improve code organization and readability, including `getPlutaValue.ts`, `getCurrentEventMultiplier.ts`, and `updatePlutaValue.ts`. [[1]](diffhunk://#diff-f6e45e9b832feab2e4649404919c110d1ab6688b87e293c82fe807f90d497888L1-R4) [[2]](diffhunk://#diff-4262c756fbdc4c4762d9403bab59991fa5d27c430160cc8c08f4a8fb0e63e0d8L1-R1) [[3]](diffhunk://#diff-2f45ba07a399e2fec4ee9070b25d3cb6d9d71a1eede7c6d8a50d5407b7c15be1L2-R10) [[4]](diffhunk://#diff-b47e5b311e120a47badff4c08f7429ed8ed6582d8fb356aa3bbb57dfb7adf4b6L2-R2) [[5]](diffhunk://#diff-1f16b70837f9bec466d186f82fff63a1544f48cc84d86be4c65a61977b9a69e8L2-L55)
* [`discordWebhook.ts`](diffhunk://#diff-0d66f0ea25c8e1ad9b2b8b561b8820ca8d0ac3d29fd5de5163f4de69fced6c01R1-R28): Added functions to send Pluta values and developer messages to a Discord webhook, improving monitoring and notifications.

### Web App Enhancements:

* [`package.json`](diffhunk://#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R17-R21): Added new dependencies for `chartjs-adapter-date-fns`, `react-chartjs-2`, and `react-router-dom` to support new features in the web app.
* [`Background.css`](diffhunk://#diff-0a0174d07beb38a02092a062044f2fb2ba7b93e6ba05c9d4cec6875ab3353a1bR7-R8): Adjusted the background styling to include opacity and blur effects for a better visual experience.

### CODE REVIEW

**Comment by Danrejk** [comment]
@Beukot please review

**Comment by Danrejk** [comment]
@Beukot 👍= help

**Review by Beukot — COMMENTED** [review]
👍

**Review by Beukot — APPROVED** [review]

File: api/src/websocket/utils/discordWebhook.ts
Comment (4o4fra) [inline api/src/websocket/utils/discordWebhook.ts:1]: PDW webhook

Comment (Danrejk) [reply api/src/websocket/utils/discordWebhook.ts:1]: pdwuuu

**Pull request merged**